Bucketeer/Sources/Internal/Model/MetricsEventData.swift

```diff
@@ -15,7 +15,7 @@ enum MetricsEventData: Hashable {
     struct GetEvaluationLatency: Codable, Hashable {
         let apiId: ApiId
         let labels: [String: String]
-        let duration: Duration
+        let duration: Float64
```

Comment:
Removed type and change it to Float64.

Reply:
We must also change the field name as described [here](https://github.com/ca-dp/bucketeer-ios-sdk/pull/92#issuecomment-1486918665).
Please also rename all the function's parameter names.

Reply:
Can you run the e2e test from there so I can verify that the Latency metrics event is working as expected before merging this PR?